src/Workers/ProcessorInterfaceAdapter.php

```diff
@@ -20,11 +21,12 @@ public function __construct(WorkerContext $workerContext)
     {
         $this->workerContext = $workerContext;
         $this->eventDispatcher = new NullEventDispatcher();
+        $this->messageAdapterFactory = null;
```

Comment:
Used in L29 ``$message = $this->createMessageAdapter($message);``
Explanation in https://github.com/puzzle-org/amqp/blob/2349818d10d230f30fe505fbb0219feeb0e075c6/src/Workers/MessageAdapterFactoryAware.php

It's a drawback of using traits :-(